### PR TITLE
New module display

### PR DESCRIPTION
**Pull request by AVRH**
New design for module display on the front page
Background colours of the modules changed to work with the new display style
ModuleSmallCourseCard colours edited to work with the new display style

### CODE REVIEW

File: frontend/components/Home/ModuleDisplay/ModuleCourseCardList.tsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        <ThreeOrLessCoursesListing courses={courses} />
+      ) : (
+        <ThreeOrLessCoursesListing courses={courses.slice(0, 3)} />
+      )}
+      <ShowMoreButton
+        fullWidth={true}
+        variant="contained"
+        onClick={() => setShowAll(!showAll)}
+      >
+        {showAll ? "Näytä rajatut" : "Näytä kaikki"}
```
Comment (nygrenh) [inline frontend/components/Home/ModuleDisplay/ModuleCourseCardList.tsx]: These are not translated.

File: frontend/components/Home/ModuleDisplay/ModuleCoursesDisplay.tsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+  return (
+    <CoursesListContainer>
+      <CoursesListTitle
+        fontcolor="black"
+        titlebackground="white"
+        variant="h3"
+        component="h3"
+      >
+        Opintokokonaisuuteen kuuluvat kurssit
```
Comment (nygrenh) [inline frontend/components/Home/ModuleDisplay/ModuleCoursesDisplay.tsx]: Not translated

**Review by nygrenh — CHANGES_REQUESTED** [review]

File: frontend/components/Home/ModuleDisplay/ModuleDisplay.tsx [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+interface ModuleProps {
+  module?: AllModules_study_modules_with_courses
+  hueRotateAngle: number
+  brightness: number
+  backgroundColor: string
+}
+
+function Module(props: ModuleProps) {
+  const { module, hueRotateAngle, brightness, backgroundColor } = props
+  //@ts-ignore
```
Comment (nygrenh) [inline frontend/components/Home/ModuleDisplay/ModuleDisplay.tsx]: Is this required?

**Review by nygrenh — CHANGES_REQUESTED** [review]

File: frontend/components/Home/ModuleDisplay/ModuleDisplay.tsx (hunk 3 above)
Comment (AVRH) [reply frontend/components/Home/ModuleDisplay/ModuleDisplay.tsx]: Leftover from development. good catch!

File: frontend/components/Home/ModuleDisplay/ModuleCoursesDisplay.tsx (hunk 2 above)
Comment (AVRH) [reply frontend/components/Home/ModuleDisplay/ModuleCoursesDisplay.tsx]: Done

File: frontend/components/Home/ModuleDisplay/ModuleCourseCardList.tsx (hunk 1 above)
Comment (AVRH) [reply frontend/components/Home/ModuleDisplay/ModuleCourseCardList.tsx]: Done

**Review by nygrenh — APPROVED** [review]

**Pull request merged**